src/commands/GameIntegration/ffxiv.ts

```diff
@@ -42,8 +42,8 @@ export default class extends RichDisplayCommand {
 		return response;
 	}
 
-	private async fetchCharacter(i18n: Language, name: string) {
-		const searchResult = await searchCharacter(i18n, name);
+	private async fetchCharacter(i18n: Language, name: string, server?: FFXIV.Server) {
```

Comment:
```suggestion
	private async fetchCharacter(i18n: Language, name: string, server?: string) {
```

Casting this to a string union serves to purpose because there is no runtime type checking